internal/cli/main.go

```diff
@@ -186,16 +186,33 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 			log.fatalErr(migraterErr)
 		}
 
-		limit := -1
-		if flag.Arg(1) != "" {
-			n, err := strconv.ParseUint(flag.Arg(1), 10, 64)
-			if err != nil {
-				log.fatal("error: can't read limit argument N")
+		downFlagSet := flag.NewFlagSet("down", flag.ExitOnError)
+		applyAll := downFlagSet.Bool("all", false, "Apply all down migrations")
+
+		args := flag.Args()[1:]
+		if err := downFlagSet.Parse(args); err != nil {
+			log.Println(err)
+		}
+
+		downArgs := downFlagSet.Args()
+		num, needsConfirm, numErr := numDownMigrationsFromArgs(*applyAll, downArgs)
```

Comment:
I don't see a point in creating a new error variable. e.g. use `err` instead of `numErr`

Reply:
switched to `err`